Octokit/Http/HttpClientAdapter.cs

```diff
@@ -70,16 +70,22 @@ static CancellationToken GetCancellationTokenForRequest(IRequest request, Cancel
 
             object responseBody = null;
             string contentType = null;
+
+            // We added support for downloading images,zip-files and application/octet-stream. 
+            // Let's constrain this appropriately.
+            var binaryContentTypes = new[] {"application/zip"
+                ,"application/x-gzip",
+                "application/octet-stream"};
```

Comment:
I don't know if the response content type could be anything else other than `application/octet-stream`. From the test's that I have done with API  for `zip` and `txt` the response content type has only been `application/octet-stream`. Do we need the other types?